Makefile

```diff
@@ -22,7 +25,7 @@ install:
 bindata:
 	which go-bindata > /dev/null || go get -u github.com/jteeuwen/go-bindata/...
 	rm -rf bootstrap/bootstrap.go
-	go-bindata -pkg bootstrap -o bootstrap/bootstrap.go bootstrap/ bootstrap/vpn
+	go-bindata -pkg bootstrap -o bootstrap/bootstrap.go bootstrap/
```

Comment:
The bootstrap/vpn was removed here. This is needed to make sure the files from this folder can be read.